src/errors.rs

```diff
@@ -23,6 +23,15 @@ error_chain!{
         MissingManifest {
             description("Unable to find Cargo.toml")
         }
+        /// Cargo.toml is valid toml, but doesn't contain the expected fields
+        InvalidManifest {
+            description("Cargo.toml missing expected `package` or `project` fields")
```

Comment:
It occurs to me that we might want to put `Cargo.toml` in back ticks. But we don't do it in the other errors just above this… Yeah, it's fine, don't change it 😅